src/scss/partials/_content-box.scss

```diff
@@ -0,0 +1,25 @@
+.content-box {
+  height: 100%;
+  padding: 200px 100px 0 100px;
```

Comment:
Added breakpoints that use `vw`s for horizontal padding. Top padding is static (per breakpoint), though, as it's a part of the design.